tests/jquery.flot.navigate.Test.js

```diff
@@ -31,18 +31,38 @@ describe("flot navigate plugin", function () {
         expect(typeof plot.zoomOut).toBe('function');
         expect(typeof plot.pan).toBe('function');
         expect(typeof plot.smartPan).toBe('function');
-
-        plot.shutdown();
     });
 
     it('shuts down the plots after use', function() {
         plot = $.plot(placeholder, [
-            []
+            [
+                [0, 0],
+                [10, 10]
+            ]
+        ], options);
+        var canvasElement = placeholder[0].childNodes[2];
+
+        var spy = spyOn(canvasElement, 'removeEventListener').and.callThrough();
+
+        plot = $.plot(placeholder, [
+            [
+                [0, 0],
+                [10, 10]
+            ]
         ], options);
 
-        plot.shutdown();
+        var touchCount = 0;
+        for(var i = 0; i < spy.calls.count(); i++){
+            if(spy.calls.argsFor(i)[0] === 'touchstart' || spy.calls.argsFor(i)[0] === 'touchmove' || spy.calls.argsFor(i)[0] === 'touchend' ) {
+                touchCount++;
+            }
+        }
+
+        expect(touchCount).toEqual(3);
 
-        expect(plot.placeholder).toBe(undefined);
+        expect(spy).toHaveBeenCalledWith('touchstart', jasmine.any(Function))
+        expect(spy).toHaveBeenCalledWith('touchmove', jasmine.any(Function));
+        expect(spy).toHaveBeenCalledWith('touchend', jasmine.any(Function));
```

Comment:
I prefer these 3 expects to remain along with expect(spy,callCount).toBe(3) rather than less readable for